scripts/scripts/testing/automations/batch/suriname.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
+from bs4 import BeautifulSoup
+import requests
+from datetime import date
+import urllib
+
+url = 'https://covid-19.sr'
+req = urllib.request.urlopen(url)
+soup = BeautifulSoup(req.read(), 'html.parser')
+
+stats = soup.select('div.stats-number.ult-responsive')
+daily = int(stats[-2]['data-counter-value'])
+
+date = str(date.today())

```

Comment:
Avoid using `date` (it shadows library module) as the variable name, proposal: use `date_str`.